### ISSUE TITLE
stylus build failed

### ISSUE THREAD

**Issue by syzer** [Swiip/generator-gulp-angular#592]
I can run generator with plain css quite ok, but it fails silently with stylus:

![screen shot 2015-06-10 at 14 36 12](https://cloud.githubusercontent.com/assets/1989646/8082523/2903fb0e-0f7e-11e5-9420-c3d73bce791b.png)
yo gulp-angular

```
[14:38:48] Warning: gulp version mismatch:
[14:38:48] Global gulp is 3.9.0
[14:38:48] Local gulp is 3.8.11
[14:38:49] Using gulpfile ~/gulp-stylus/gulpfile.js
[14:38:49] Starting 'clean'...
[14:38:49] Finished 'clean' after 43 ms
[14:38:49] Starting 'default'...
[14:38:49] Starting 'scripts'...
[14:38:49] Starting 'styles'...
[14:38:49] Starting 'partials'...
[14:38:49] Starting 'fonts'...
[14:38:49] Starting 'other'...
[14:38:49] Finished 'default' after 610 ms
[14:38:49] gulp-inject Nothing to inject into index.styl.
[14:38:49] all files 3.26 kB
[14:38:49] Finished 'scripts' after 891 ms
[14:38:49] Finished 'styles' after 751 ms
[14:38:49] Starting 'inject'...
[14:38:50] gulp-inject 1 files into index.html.
[14:38:50] gulp-inject 3 files into index.html.
[14:38:50] Finished 'partials' after 589 ms
[14:38:50] Finished 'inject' after 134 ms
[14:38:50] Starting 'html'...
[14:38:50] gulp-inject 1 files into index.html.
[14:38:50] Finished 'fonts' after 1.02 s
[14:38:59] 'dist/' styles/app-4b91b08d.css 130 B
[14:38:59] 'dist/' scripts/app-4d84109a.js 3.59 kB
[14:38:59] Finished 'other' after 9.52 s
[14:38:59] 'dist/' scripts/vendor-5ae9a0f5.js 341.82 kB
[14:38:59] 'dist/' index.html 1.06 kB
[14:38:59] 'dist/' all files 346.6 kB
[14:38:59] Finished 'html' after 9.1 s
[14:38:59] Starting 'build'...
[14:38:59] Finished 'build' after 3.65 μs
```
```
[14:39:15] Warning: gulp version mismatch:
[14:39:15] Global gulp is 3.9.0
[14:39:15] Local gulp is 3.8.11
[14:39:16] Using gulpfile ~/gulp-stylus/gulpfile.js
[14:39:16] Starting 'scripts'...
[14:39:16] Starting 'styles'...
[14:39:16] gulp-inject Nothing to inject into index.styl.
[14:39:17] all files 3.26 kB
[14:39:17] Finished 'scripts' after 687 ms
[14:39:17] Finished 'styles' after 537 ms
[14:39:17] Starting 'inject'...
[14:39:17] gulp-inject 1 files into index.html.
[14:39:17] gulp-inject 3 files into index.html.
[14:39:17] Finished 'inject' after 101 ms
[14:39:17] Starting 'watch'...
[14:39:17] Finished 'watch' after 15 ms
[14:39:17] Starting 'serve'...
[14:39:17] Finished 'serve' after 17 ms
[BS] [BrowserSync SPA] Running...
[BS] Access URLs:
 ----------------------------------------
       Local: http://localhost:3001/
    External: http://192.168.80.126:3001/
 ----------------------------------------
          UI: http://localhost:3002
 UI External: http://192.168.80.126:3002
 ----------------------------------------
[BS] Serving files from: .tmp/serve
[BS] Serving files from: src
```

**Comment by zckrs** [comment]
:tada: Just released v0.12.0 :tada: 

Can you retry with last version ?

**Comment by zckrs** [comment]
Fixed by [v0.12.1](https://github.com/Swiip/generator-gulp-angular/releases/tag/v0.12.1)
